docs/docusaurus/docs/guides/setup/configuring_data_contexts/components_how_to_configure_a_new_data_context_with_the_cli/_initialize_data_context_with_the_cli.mdx

```diff
@@ -1,36 +0,0 @@
-import TechnicalTag from '@site/docs/term_tags/_tag.mdx';
```

Comment:
The error is related to this file removal. I believe that this may be due to downloading the old code. 

docs/prepare_to_build_docs.sh
```
for version in $VERSIONS; do
  print_orange_header "Copying code referenced in docs from $version and writing to versioned_code/version-$version"

  curl -LJO "https://github.com/great-expectations/great_expectations/archive/refs/tags/$version.zip"
  unzip -oq "great_expectations-$version.zip" -d versioned_code
  mv versioned_code/"great_expectations-$version" versioned_code/version-"$version"
  rm "great_expectations-$version.zip"

done
```

Old code would continue to contain ref to this file. But docusaurus seems to search for the file in current code & not in old code.

@anthonyburdi Does my conclusion make sense to you?

Reply:
Awesome!! I see you made the regex take any import, not just the TechnicalTag. Makes sense to me! Thank you for addressing this!

One minor nit is to also change these function names & corresponding docstrings to also be more general:
- `use_relative_imports_for_tag_references`
- `_use_relative_imports_for_tag_references_substitution` 
- `_use_relative_imports_for_tag_references_substitution_path_starting_with_forwardslash`

Maybe to something like:
- `use_relative_path_for_imports`
- `_use_relative_path for import_substitution` 
- `_use_relative_path_for_import_substitution_path_starting_with_forwardslash`